tests/runtime/engine/runner.py

```diff
@@ -262,7 +266,9 @@ def run_test(test):
                     attached = True
                     signal.alarm(test.timeout or DEFAULT_TIMEOUT)
                     if test.after:
-                        after = subprocess.Popen(test.after, shell=True, start_new_session=True)
+                        after = subprocess.Popen(test.after, shell=True, start_new_session=True,
+                                                 stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
The other shellouts we do we combine stdout and stderr into one stream. I typically find that helpful cuz a lot of programs will interleave stdout and stderr in a way where separating them makes it harder to understand. WDYT about combining the streams here too?